pages/dkp/kaptain/1.3.0/download/index.md

```diff
@@ -31,5 +29,7 @@ Kaptain install package placed in the local directory. See install instructions
 
 You see the `kubeflow-1.4.0_1.3.0.tgz` file in your working directory. Follow the [Kaptain installation instructions][install] using these binaries. If you have problems downloading or installing Kaptain, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
 
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.3.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z) and [https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z](https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z).
```

Comment:
Updating with the text from the files in the [joint docs folder](https://drive.google.com/drive/u/0/folders/1eb6tnAx6Il9YvyVKZ2Oz0Scubhuwis7P)
```suggestion
This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.3.0 are available at these URLs:
* [https://github.com/minio/operator/releases/tag/v4.0.3](https://github.com/minio/operator/releases/tag/v4.0.3)
* [https://github.com/minio/mc/releases/tag/RELEASE.2021-03-23T05-46-11Z](https://github.com/minio/mc/releases/tag/RELEASE.2021-03-23T05-46-11Z)
```